app/components/Sidebar/components/CollectionLink.tsx

```diff
@@ -278,27 +273,30 @@ function CollectionLink({
         {displayDocumentLinks && manualSort && (
           <DropCursor isActiveDrop={isOverReorder} innerRef={dropToReorder} />
         )}
-        {isDraggingAnyCollection && (
-          <DropCursor
-            isActiveDrop={isCollectionDropping}
-            innerRef={dropToReorderCollection}
-          />
-        )}
-      </div>
-      {displayDocumentLinks &&
-        collectionDocuments.map((node, index) => (
-          <DocumentLink
-            key={node.id}
-            node={node}
-            collection={collection}
-            activeDocument={activeDocument}
-            prefetchDocument={prefetchDocument}
-            canUpdate={canUpdate}
-            isDraft={node.isDraft}
-            depth={2}
-            index={index}
-          />
-        ))}
+      </Relative>
+      <Relative>
+        <Folder $expanded={displayDocumentLinks}>
```

Comment:
This change worries me a bit because there are teams with thousands of documents, this will now render them hidden wherase before it didn't at all.

Reply:
good point, can probably be solved by making a distinction between "collapsed because user said so" and "collapsed because in the middle of a drag"

it's still not perfect behavior that way but performance will be optimized